templates/functions/src/queue-processor.functions.ts

```
@@ -0,0 +1,12 @@
+import { pikkuSessionlessFunc } from '../.pikku/pikku-types.gen.js'
+
+export const queueProcessor = pikkuSessionlessFunc<
+  { message: string; fail: boolean },
+  { result: string }
+>(async ({}, data) => {
```

Biome (1.9.4)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)